src/connectors/smart-sort/connectSmartSort.ts

```diff
@@ -98,6 +82,7 @@ const connectSmartSort: SmartSortConnector = function connectSmartSort(
             typeof appliedRelevancyStrictness !== 'undefined' &&
             appliedRelevancyStrictness > 0 &&
             appliedRelevancyStrictness <= 100,
+          isVirtualReplica: typeof appliedRelevancyStrictness !== 'undefined',
```

Comment:
```suggestion
          isVirtualReplica:appliedRelevancyStrictness !== undefined,
```

you can do an assertion directly against undefined